README.md

```diff
@@ -14,6 +14,21 @@ The script can install, remove, and check the status of an installed Salt minion
 either using a direct command line option or via VMware's use of Guest Variables,
 commonly referred to as guestVars.
 
+In every two-step installation example, you would be well-served to **verify against the SHA256
+sum** of the downloaded ``svtminion.sh`` file.
+
+.. _sha256sums:
```

Comment:
```suggestion
## sha256sums
```

Suggested update from `rST` to Markdown.

Not sure if Markdown usage impacts anything like the functionality of `.github/workflows/scripts/update-release-shasum.py`

Reply:
using the actions from bootstrap-salt, and it looks for the .. _sha256sums:, will change the python code in update-release-shasum.py to look for ```## sha256sums``` and update the README.md for that marker